src/plugins/data/common/field_formats/converters/percent.ts

```diff
@@ -19,20 +19,18 @@
 
 import { createNumeralFormat } from './_numeral';
 
-export function createPercentFormat() {
-  return createNumeralFormat({
-    id: 'percent',
-    title: 'Percentage',
+export const PercentFormat = createNumeralFormat({
```

Comment:
I think we should remove createNumeralFormat method at all. In this case we can extend PercentFormat from NumberFormat

Reply:
For better perception the `NumeralFormat` has been removed.

Reply:
I see now PercentFormat extended from FieldFormat. It should be extended from NumeralFormat or NumberFormat